draft-ietf-rats-architecture.md

```diff
@@ -376,6 +376,76 @@ create Evidence. Examples of environments that can be used as
 Attesting Environments include Trusted Execution Environments (TEE),
 embedded Secure Elements (eSE), or Hardware Security Modules (HSM).
 
+## Layered Attestation Procedures
```

Comment:
I think a picture is worth 1000 words. Dave T had an ascii art example that seemed pretty good. Is there a reason not to include it here?

Reply:
Let's have look and maybe a PR.